src/Agent.Worker/ExecutionContext.cs

```diff
@@ -573,6 +580,17 @@ public void InitializeJob(Pipelines.AgentJobRequestMessage message, Cancellation
             _jobServerQueue.JobServerQueueThrottling += JobServerQueueThrottling_EventReceived;
         }
 
+        private string GetWorkspaceIdentifier(Pipelines.AgentJobRequestMessage message)
+        {
+            message.Variables.TryGetValue("system.collectionId", out VariableValue collectionIdVar);
+            message.Variables.TryGetValue("system.definitionId", out VariableValue definitionIdVar);
```

Comment:
Please define constants for these in `src\Microsoft.VisualStudio.Services.Agent\Constants.cs` under `Variables`.

Reply:
they were already defined. I meant to fix this before pushing but forgot.